### PR TITLE
Created user.py model and user_requests.py

### PR DESCRIPTION
**Pull request by lechrsty**
Let me know how this looks!

### CODE REVIEW

**Comment by rickycanepa** [comment]
sucks

File: views/user_requests.py [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+        """)
+
+        users = []
+
+        dataset = db_cursor.fetchall()
+
+        for row in dataset:
+
+                user = User(row['id'], row['first_name'], row['last_name'], row['email'],
+                                row['bio'], row['username'], row['password'], row['profile_image_url'],row['created_on'], row['active'])
```
Comment (Jordon-Perkins) [inline views/user_requests.py:64]: row['active'] is not indented properly

**Review by rickycanepa — CHANGES_REQUESTED** [review]
No need for list of user dictionaries

File: views/user_requests.py (hunk 1 above)
Comment (lechrsty) [reply views/user_requests.py:64]: you trippin

**Review by rickycanepa — COMMENTED** [review]
Good to merge

**Pull request closed without merging**